elastic-buffer-sim/tests/Tests/Bittide/Simulate.hs

```diff
@@ -31,26 +30,15 @@ tests = testGroup "Simulate"
     ]
   ]
 
-fastPeriod :: PeriodPs
-fastPeriod = hzToPeriod 200e6
-
-clockConfig :: Ppm -> ClockControlConfig 7
-clockConfig clockUncertainty = ClockControlConfig
-  { cccPessimisticPeriod = speedUpPeriod clockUncertainty fastPeriod
-  , cccSettlePeriod      = fastPeriod * 200
-  , cccDynamicRange      = clockUncertainty * 2
-  , cccStepSize          = 10
-  , cccBufferSize        = d7 -- 2**7 ~ 128
-  }
-
 case_clockControlMaxBound :: Assertion
 case_clockControlMaxBound = do
   let
-    config = clockConfig (Ppm 100)
+    config = defClockConfig
     dataCounts = pure maxBound :> Nil
     changes =
       sampleN
-        (fromIntegral (cccPessimisticPeriod config))
+        -- +100 assumes callisto's pipeline less than 100 deep
+        (fromIntegral (cccPessimisticSettleCycles config + 100))
```

Comment:
Sounds like asking for trouble, is it a good idea to add a check that checks the pipeline depth?

Reply:
Are you happy if I replace it with 10_000?

Reply:
I'll allow it